eth/package.json

```diff
@@ -20,6 +20,7 @@
     "@projectsophon/workspace": "^2.0.0",
     "@solidstate/contracts": "^0.0.41",
     "@solidstate/hardhat-4byte-uploader": "^1.0.2",
+    "@solidstate/spec": "^0.0.41",
```

Comment:
Are you actually using this somewhere to implement in the tests?

Reply:
Not anymore, I'll remove it.